modules/standard/Random.chpl

```diff
@@ -566,6 +574,10 @@ module Random {
      imag, and complex numbers; and generating numbers in a range in parallel.
      These features are not available in the reference implementations of PCG.
 
+     .. note::
+
+       This interface provided by this module is expected to change.
```

Comment:
Really minor but "The interface" scans better for me... not worth it unless you've got other changes to make, though